ui/app/AppLayouts/Profile/views/ChangePasswordView.qml

```diff
@@ -25,15 +25,20 @@ SettingsContentBase {
 
     readonly property bool biometricsEnabled: localAccountSettings.storeToKeychainValue === Constants.keychain.storedValue.store
 
+    readonly property StatusSwitch biometricsSwitch: titleRowComponentLoader.item.switchItem
     readonly property Connections privacyStoreConnections: Connections {
         target: Qt.platform.os === Constants.mac ? root.privacyStore.privacyModule : null
 
         function onStoreToKeychainError(errorDescription: string) {
-            biometricsSwitch.checked = Qt.binding(() => { return biometricsSwitch.currentStoredValue })
```

Comment:
@alexandraB99 could you please clarify the scope of this this past line if you remember? `currentStoredValue` doesn't exist anymore. I can't seem to find what wha the purpose of this new binding.